pkg/uotel/config.go

```
@@ -0,0 +1,513 @@
+			return nil, fmt.Errorf("otel: failed to decode base64 TLS certificate: %w", err)
+		}
+		certData = decodedCert
+	}
+
+	// Create a certificate pool and add the certificate
+	certPool := x509.NewCertPool()
+	if ok := certPool.AppendCertsFromPEM(certData); !ok {
+		return nil, fmt.Errorf("otel: failed to parse TLS certificate")
+	}
+
+	return &tls.Config{
```

golangci-lint (1.64.8)

[high] G402: TLS MinVersion too low.

(gosec)